### PR TITLE
multiple ports: quoting

### PR DESCRIPTION
**Pull request by l2dy**
#### Description
Most ports and our guide use `${prefix}` rather than `$prefix`.

### CODE REVIEW

**Comment by stevecheckoway** [comment]
LGTM

**Comment by neverpanic** [comment]
LGTM for my ports.

**Comment by blair** [comment]
LGTM for my ports.

**Comment by MarcusCalhoun-Lopez** [comment]
This change looks fine for my ports.

**Comment by rowue** [comment]
LGTM for my Port

**Comment by michaelld** [comment]
LGTMT

File: editors/cedet/Portfile [hunk 1]
```diff
@@ -48,8 +48,8 @@ destroot {
         global  destroot worksrcpath
         upvar   lispdir lispdir
 
-        xinstall -d $destroot$lispdir/$package_dir
-        foreach f [glob -directory $worksrcpath/$package_dir *] {
+        xinstall -d ${destroot}$lispdir/$package_dir
+        foreach f [glob -directory ${worksrcpath}/$package_dir *] {
```
Comment (pmetzger) [inline editors/cedet/Portfile]: Surprised you didn't do "lispdir" and "package_dir" while you were here. (See also the rest of this file.)

File: ocaml/ocaml-uint/Portfile [hunk 2]
```diff
@@ -21,7 +21,7 @@ use_oasis           yes
 use_oasis_doc       yes
 
 patch {
-    reinplace "s|\$prefix/include|${destroot}\$prefix/include|g" ${worksrcpath}/setup.ml
+    reinplace "s|\${prefix}/include|${destroot}\${prefix}/include|g" ${worksrcpath}/setup.ml
```
Comment (pmetzger) [inline ocaml/ocaml-uint/Portfile]: Given the quoting of `$` here, I suspect this is incorrect.

**Comment by pmetzger** [comment]
I found a couple of problems, one is non-substantive and one is substantive.

File: editors/cedet/Portfile (hunk 1 above)
Comment (l2dy) [reply editors/cedet/Portfile]: Thanks. I did a global search and replace, which is also why I missed the quoted `$prefix`. Both fixed.

**Pull request merged**